springwolf-add-ons/springwolf-generic-binding/src/test/java/io/github/springwolf/addons/generic_binding/annotation/processor/AsyncGenericOperationBindingProcessorTest.java

```diff
@@ -101,6 +100,30 @@ void twoPropertiesTest() {
             assertThat(result).isEqualTo(Map.of("key1", "value1", "key2", "value2"));
         }
 
+        @Test
+        void arrayPropertyTest() {
+            // given
+            String[] strings = {"key=[value1, value2, value3]"};
```

Comment:
I read about the Java properties format and the native format is basically a key/value map of strings without support for further data types.
the closed about arrays/sequences that I found in the context of Spring is https://stackoverflow.com/a/33944154 which suggests the following format: ˋkey=value1, value2, value3ˋ (no square brackets)

Does this makes sense here?

Reply:
I saw it too, but it doesn't seem to work. Looking at java.util.Properties (which is what we use for the parsing) it doesn't seem to support arrays out-of-box

Reply:
My main concern is that removing the square brackets makes the parsing harder.

`{"array=1,2,3", "description=something, something else"}`

The first should be an array, while the second should be a string, but removing the `[]` it becomes more difficult to guess how should be parsed.

Reply:
Good point! I guess the square brackets around the values is the most natural approach.

Will consult with @sam0r on Friday